src/CommonITILObject.php

```diff
@@ -520,7 +520,12 @@ protected function setPredefinedFields(ITILTemplate $tt, array &$options, array
                         ) {
                             $options[$predeffield]           = $predefvalue;
                             $predefined_fields[$predeffield] = $predefvalue;
-                            $this->fields[$predeffield]      = $predefvalue;
+                            if (
+                                ($this->isNewItem())
+                                && (!array_key_exists($predeffield, $this->fields) || empty($this->fields[$predeffield]))
```

Comment:
Some fields have default non-empty values. Wouldn't this make them impossible to predefine?

Example,
internal time to own is a string with the text "NULL"
urgency and impact are integers with the value 3
priority is a string "3" (No idea why it is a different type)

Reply:
Hum, it was changed in #10886.

Reply:
nope

Reply:
Ok, so I guess, to restore the behaviour from GLPI 9.5, we should remove the `$this->getType() == "Ticket" ||` condition on line 495, and remove condition added by current PR..